apps/measures/templates/apps_measures/measures_detail_page.html

```diff
@@ -0,0 +1,97 @@
+                {{ page.responsible_orgs }}
+            </dd>
+        </div>
+        <div class="meta-information__column">
+            <div class="meta-information__column__head">
+                {% trans 'District' %}
+            </div>
+            <div class="meta-information__column__content">
+                {% with districts=page.get_districts_display %}
+                    {{ districts }}
+                {% endwith %}
+            </div>
+        </div>
+    </dl>
+    <dl class="meta-information__row">
+        <div class="meta-information__column">
+            <dt class="meta-information__column__head">
+                {% trans 'Involved organisations' %}
+            </dt>
+            <dd class="meta-information__column__content">
+                {{ page.involved_orgs }}
+            </dd>
+        </div>
+        <div class="meta-information__column">
+            <dt class="meta-information__column__head">
+                {% trans 'Period' %}
+            </dt>
+            <dd class="meta-information__column__content">
+                {{ page.date }}
+            </dd>
+        </div>
+    </dl>
+    <dl class="meta-information__row">
+        <div class="meta-information__column">
+            <dt class="meta-information__column__head">
+                {% trans 'Handlungsfelder' %}
+            </dt>
+            <dd class="meta-information__column__content">
+                tbd
+            </dd>
+        </div>
+        <div class="meta-information__column">
+            <dt class="meta-information__column__head">
+                {% trans 'Status' %}
+            </dt>
+            <dd class="meta-information__column__content">
+                {{ page.get_status_display }}
+            </dd>
+        </div>
+    </dl>
+    {% image page.text_image original as text_image %}
+    <div class="meta-information__image">
+        <img
+            src="{{ text_image.url }}"
+            alt="{{ text_image.alt }}"
+        />
+        {% if text_image.caption %}
+            <p class="image__caption">{{ text_image.caption }}</p>
+        {% endif %}
+    </div>
+    {{ page.body|richtext }}
+    {% if page.body_participation %}
+        <h2>{% trans 'Participation' %}</h2>
+        {{ page.body_participation|richtext }}
+    {% endif %}
+    {% if page.body_effect %}
+        <h2>{% trans 'Effect' %}</h2>
+        {{ page.body_effect|richtext }}
+    {% endif %}
+    <h2>{% trans 'Contact' %}</h2>
+    <h3>{% trans 'Organisation' %}</h3>
+    {{ page.contact_organisation_name }}
+    <h3>{% trans 'Contact person' %}</h3>
+    {{ page.contact_name }}<br />
+    {% if page.contact_email %}
+        {% trans 'Email' %}: {{ page.contact_email }}<br />
+    {% endif %}
+    {% if page.contact_phone %}
+        {% trans 'Phone number' %}: {{ page.contact_phone }}<br />
+    {% endif %}
+    {% if page.contact_website %}
+        {% trans 'Website' %}: {{ page.contact_website }}
+    {% endif %}
```

Comment:
I would wrap the contact deets in an `<address>` element.
They should also probably be links as makes it more accessible but guess we check with Marcelo first.